14/umbraco-cms/reference/configuration/modelsbuildersettings.md

```diff
@@ -32,6 +32,10 @@ Specifies how the models builder will generate models and when to generate them.
 * `SourceCodeManual` - Models will be generated as `.cs` files whenever a user clicks the "Generate models" button on the models builder dashboard - however, the models will not be compiled and loaded into memory dynamically. This means that models are available to edit within the project. The project needs to be recompiled and restarted for the new models, or model changes, to take effect.
 * `SourceCodeAuto` - This mode behaves the same as `SourceCodeManual` with one difference, the generation of models happens automatically every time a content type change occurs.
 
+{% hint style="code" %}
```

Comment:
```suggestion
{% hint style="info" %}
```

Reply:
Hi Sofie! 

The local one is normally a copy you can make yourself from Development - after that, you can rename it.

Reply:
Alright. Should we link off to some docs about how to do this, you think?
Also, I remember config transforms being a Cloud thing - but it's also used for CMS now?